velox/functions/prestosql/aggregates/BitwiseAggregates.cpp

```diff
@@ -188,10 +191,10 @@ bool registerBitwiseAggregate(const std::string& name) {
   return true;
 }
 
-static bool FB_ANONYMOUS_VARIABLE(g_AggregateFunction) =
-    registerBitwiseAggregate<BitwiseOrAggregate>(kBitwiseOr);
-static bool FB_ANONYMOUS_VARIABLE(g_AggregateFunction) =
-    registerBitwiseAggregate<BitwiseAndAggregate>(kBitwiseAnd);
+void registerBitwiseAggregate() {
```

Comment:
registerBitwiseAggregate -> registerBitwiseAggregates with an 's' at the end for plural (this function registers multiple aggregates, not one)